codalab/worker_manager/worker_manager.py

```diff
@@ -247,3 +252,19 @@ def run_one_iteration(self):
             logger.info('Starting a worker!')
             self.start_worker_job()
             self.last_worker_start_time = time.time()
+
+    def filter_bundles(
```

Comment:
This is a much needed change---thanks! Is it worth logging the bundles that get filtered out? I could see a scenario where someone launches a worker manager that doesn't have sufficient resources for a staged bundle, and the user is confused why the worker manager isn't launching jobs.